tests/integration/tech-record-document-conversion.intTest.ts

```diff
@@ -0,0 +1,153 @@
+import {StartedTestContainer} from "testcontainers";
+import {destroyConnectionPool, executeSql} from "../../src/services/connection-pool";
+import {exampleContext, useLocalDb} from "../utils";
+import techRecordDocumentJson from "../resources/dynamodb-image-technical-record.json";
+import {getContainerizedDatabase} from "./cvsbnop-container";
+import {TechRecordUpsertResult} from "../../src/models/upsert-results";
+import {processStreamEvent} from "../../src/functions/process-stream-event";
+import {getConnectionPoolOptions} from "../../src/services/connection-pool-options";
+
+useLocalDb();
+
+describe("convertTechRecordDocument() integration tests", () => {
+    let container: StartedTestContainer;
+
+    beforeAll(async () => {
+        jest.setTimeout(60_000);
```

Comment:
This is an interesting one and not sure if optimisation could be required in the future. It is not something we can really control but I guess it didn't time out in the pipeline and built the service already?
I think some good points/questions have been raised earlier around stress/perf testing. Do you know what a prod record would look like?
We might ask Mike or Sanj but hopefully we won't expect tech-records or test-results to be to big

Reply:
Hi, this is not the time it takes update-store to process one document. That time is about 7 seconds on average from the time the stream event is emitted from the source database to the time it reaches the destination database. This `60_000` ms is the time it takes for the docker daemon to spin up a MySQL image and for the spawned Liquibase process to apply to it. the test itself takes less than 300ms on average :)

Reply:
With respect to records similar to production - see some examples based on Deepika's regression suite under my own `src/tests/resources`:
* HGV, PSV and TRL tech records
* HGV, PSV and TRL test results